tests/modules/__init__.py

```diff
@@ -25,7 +25,8 @@
 
 __temp_example_images_folder = None
 
-cp_logo_url = "https://raw.githubusercontent.com/CellProfiler/CellProfiler/master/artwork/CP_logo.png"
+# FIXME: use a stable URI
+cp_logo_url = "https://raw.githubusercontent.com/CellProfiler/CellProfiler/0c64193dd9108934400494fffa41d24f3df1573c/artwork/CP_logo.png"
```

Comment:
Unfortunately, it expects the shape `(70, 187, 3)`:

https://github.com/CellProfiler/CellProfiler/pull/2806/files#diff-8e024a4647cd7ab59604495f60825c9dL30